packages/schematics/src/command-line/shared.ts

```diff
@@ -385,10 +387,14 @@ export function allFilesInDir(dirName: string): string[] {
         return;
       }
       try {
+        const s = fs.statSync(child);
         if (!fs.statSync(child).isDirectory()) {
```

Comment:
This can also be changed to `s`
```suggestion
        if (!s.isDirectory()) {
```